docs/source/extension/extension_tutorial.rst

```diff
@@ -457,6 +457,16 @@ And update the ``activate`` method to be ``async`` since we are now using
 
         activate: async (app: JupyterFrontEnd, palette: ICommandPalette) =>
 
+.. note::
+
+    If you are new to JavaScript / TypeScript and want to learn more about ``async``, ``await``,
+    and ``Promises``, you can check out the following tutorial on MDN:
+    https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Promises
```

Comment:
Is it common to display a link fully or is it better to change it to:

```suggestion
    and ``Promises``, you can check out the following `tutorial on MDN <https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Promises>`_
```

Reply:
Right, let's switch to not show the full URL so it's consistent with the other links (for example the "Opening the Web Console in Firefox" link above).